tests/lc2st_test.py

```diff
@@ -0,0 +1,217 @@
+# This file is part of sbi, a toolkit for simulation-based inference. sbi is licensed
+# under the Affero General Public License v3, see <https://www.gnu.org/licenses/>.
+
+from __future__ import annotations
+
+import pytest
+import torch
+from sklearn.neural_network import MLPClassifier
+
+from sbi.diagnostics.lc2st import LC2ST, LC2ST_NF
+from sbi.inference import SNPE
+from sbi.simulators.gaussian_mixture import GaussianMixture
+
+
+@pytest.mark.parametrize("method", (LC2ST, LC2ST_NF))
+@pytest.mark.parametrize("classifier", ('mlp', 'rf', 'custom'))
+@pytest.mark.parametrize("cv_folds", (1, 2))
+def test_running_lc2st(method, classifier, cv_folds):
+    """Tests running inference and then LC2ST-(NF) and then getting test quantities."""
+
+    n_train = 100
```

Comment:
pleaser use similar names as in other tests, e.g., `num_simulations`, `num_{cal, eval, trials_null}`